tests/unittest/_torch/modeling/test_modeling_gemma4_unified.py

```
@@ -205,6 +207,14 @@ def test_get_model_architecture_resolves_wrapper():
     assert arch == _UNIFIED_ARCH
 
 
+def test_wrapper_rejects_missing_image_token_id():
+    config = Gemma4UnifiedConfig(image_token_id=None)
+    model_config = ModelConfig(pretrained_config=config)
+
+    with pytest.raises(ValueError, match="requires config.image_token_id"):
```

Ruff (0.15.21)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)